tests/integration/kubernetes/gha-run.sh

```diff
@@ -395,8 +395,8 @@ function cleanup_kata_deploy() {
 	esac
 
 	# shellcheck disable=2086
-	kubectl delete ${deploy_spec}
-	kubectl -n kube-system wait --timeout=10m --for=delete -l name=kata-deploy pod
+	kubectl_retry delete ${deploy_spec}
+	kubectl_retry -n kube-system wait --timeout=10m --for=delete -l name=kata-deploy pod
```

Comment:
In case of a legit fail, e.g., due a bug the kata-deploy pod didn't get deleted this line will take 30"15' to return (vs the original 10"). Maybe don't retry here? or if retry then decrease the original timeout?

Reply:
Oh, I missed it. Let's not use `kubectl_retry` for `kubectl` which  uses an `--timeout` option. Thanks.